<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insert_s1</class>
 <widget class="QDialog" name="insert_s1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="textEdit_orderNo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OrderNo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit_itemName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ItemName</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit_review">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_confirm">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rating</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Content</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
